version.go

```diff
@@ -53,10 +53,21 @@ func Version() string {
 
 // UserAgent returns the full user agent string that identifies the software
 // that is submitting swaps to the loop server.
-func UserAgent() string {
+func UserAgent(initiator string) string {
+	// We'll only allow "safe" characters in the initiator portion of the
+	// user agent string and spaces only if surrounded by other characters.
+	initiatorAlphabet := semanticAlphabet + ". "
+	cleanInitiator := normalizeVerString(
```

Comment:
Could also have a max length for the `initiator` (but since it's enforced on the server side, it's not super important).

Reply:
Yeah, makes sense. We don't want an UI to interfere with the success of submitting a swap. I limited the initiator part to 150 characters.